### PR TITLE
Mostieri/nexus ansys versioning

### PR DESCRIPTION
**Pull request by mariostieriansys**
As a requirement to support multiple versions of EnSight/PyEnSight on AnsysLab/OnScale Notebook we have the need to version the ansys/nexus folder in the nexus### distribution.

This PR reflects the changes needed in PyADR once the EnSight change is merged.

A fallback is provided if the version of ADR being used is < 242

### CODE REVIEW

**Review by margalva — DISMISSED** [review]
In adr_report there's a wrapper around the export as html capability. We should probably add there as well the ability to pass the ansys_version, in case someone runs pydynamicreporting on top of an older ansys installation. 
So inside export_html() you have self.service.serverobj.export_report_as_html()
Add to that method an optional ansys_version that comes from self.service._ansys_version, and eventually gets passed into the ReportDownloadHTML definition. I think this is the best approach, if you agree.

File: src/ansys/dynamicreporting/core/utils/report_remote_server.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ class Server:
     Implements REST protocols.
     """
 
-    def __init__(self, url=None, username=None, password=None):
+    def __init__(self, url=None, username=None, password=None, ansys_version=None):
```
Comment (randallfrank) [inline src/ansys/dynamicreporting/core/utils/report_remote_server.py:106]: I'm ok with this for now, but I would prefer updating the ItemAPIVersionAPIView REST endpoint to return the Ansys version number.   With that change, the Server instance will just "get" the version number.

**Review by randallfrank — APPROVED** [review]

File: src/ansys/dynamicreporting/core/utils/report_remote_server.py (hunk 1 above)
Comment (mariostieriansys) [reply src/ansys/dynamicreporting/core/utils/report_remote_server.py:106]: we will merge for now

File: src/ansys/dynamicreporting/core/utils/report_download_html.py [hunk 2]
```diff
@@ -329,9 +362,9 @@ def _get_file(self, path_plus_queries: str, pathname: str, inline: bool = False)
                         # we need to prefix the .bin file and scene.js file with the GUID
                         basename = f"{os.path.basename(os.path.dirname(pathname))}_{basename}"
                     else:
-                        tmp = self._fix_viewer_component_paths(basename, tmp)
+                        tmp = self._fix_viewer_component_paths(basename, tmp, self._ansys_version)
```
[the remark this answers is not in the document]
Comment (mariostieriansys) [reply src/ansys/dynamicreporting/core/utils/report_download_html.py:365]: we will merge for now

**Pull request merged**